lib/opportunity/opportunity.c

```diff
@@ -68,11 +71,11 @@ void OP_set_mock_random(opportunity_t *self, bool doMock)
 }
 
 /**
- * void OP_process(opportunity_t *self, uint16_t *input, uint16_t *output, uint16_t reset)
+ * static void _OP_process_reset(opportunity_t *self, uint16_t *reset)
```

Comment:
Why does the function have a comment that simply re-states the function spec?

Reply:
Lol this was a formatting thing that either you or I put into .c files and I just went with it. Maybe we just have the description of the parameters eventually. I'll delete them